.github/workflows/master-push.yml

```diff
@@ -46,17 +46,17 @@ jobs:
         key: pydicom-testdata
     - name: Initial test environment setup
       run: |
-        python -m pip install wheel 
+        python -m pip install wheel
         python -m pip install pytest pytest-cov
     - name: Test with no external libraries
       run: |
-        pytest 
+        pytest --cov=pydicom --cov-append
     - name: Install numpy
```

Comment:
Perhaps I am confused, but is this not running coverage for all matrix combinations?  We should only need it for one combination - that was the purpose of the previously named `coverage-args`.

Reply:
Yeah, I wasn't completely sure how to do that with the setup in this one (keep getting errors) so went with something that worked rather than keep experimenting.

Reply:
Ah, misunderstood you point. I'll fix it

Reply:
In the end it was simpler to add another build just for the coverage. It might be more straight-forward to add another workflow just for coverage.

 I also changed the GDCM caching so it gets cached for each python version, which I think is correct (?)

Reply:
See my comment on that line...

Reply:
I'm seeing 

```
    - name: Test with no external libraries
      run: |
        pytest ${{ matrix.pytest-args }}
```